src/main/scala/com/bwsw/cloudstack/vault/server/Components.scala

```diff
@@ -77,9 +77,11 @@ class Components(settings: Components.Settings) {
     settings.cloudStackVaultControllerSettings
   )
 
+  //event handling
   lazy val consumer = new Consumer[String, String] (
     settings.consumerSettings
   )
+  lazy val eventMapper = new JsonMapper(ignoreUnknownProperties = true)
 
   def close(): Unit = {
```

Comment:
closeFunctionList.foreach(func => {
      Try(func()) match {
        case Failure(e: Throwable) =>
          logger.error(s"the function: '$func' was executed with exception: $e")
      }
    })

Reply:
ok